src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -6480,7 +6480,7 @@ private StatementSyntax ParseStatementNoDeclaration(bool allowAnyExpression)
                         }
                         else
                         {
-                            return this.ParseLocalDeclarationStatement(parseAwaitKeyword());
+                            return this.ParseLocalDeclarationStatement(parseAwaitKeyword(MessageID.None));
```

Comment:
Interesting, so the optional parameter prevented this from being inlined?

Reply:
I suspected it was the conversions to `Nullable<MessageID>` rather than the optional parameter. I decided to make the parameter non-optional at the same time.

---
In reply to: [356841763](https://github.com/dotnet/roslyn/pull/40290#discussion_r356841763) [](ancestors = 356841763)

Reply:
Makes sense.